asciidoctor-revealjs.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.summary = 'Converts AsciiDoc to HTML for a Reveal.js presentation'
   s.description = 'Reveal.JS back-end for Asciidoctor and Asciidoctor.js. Write slides in AsciiDoc!'
   s.license = 'MIT'
-  s.required_ruby_version = '>= 2.3.8'
+  s.required_ruby_version = '>= 2.3.0'
```

Comment:
My preference is to not set the required Ruby version, just document it in the README. The reason is, there's no way to override the required Ruby version at install time, and this can break builds. If you disagree, that's fine. I'm just sharing what I do.

Reply:
Fixed with 4fce207. Followed your advice.